README.md

```diff
@@ -19,7 +16,7 @@ Redundant information in overlapping Bragg disks can be leveraged to calculate t
 Structure in the diffracted halos of amorphous systems can be used to describe the short and medium range order.
 
 
-py4DSTEM supports many different modes of 4DSTEM analysis.
+**py4DSTEM** supports many different modes of 4DSTEM analysis.
```

Comment:
I'm not sure if I like having py4DSTEM be bold everywhere?  Can we just do this once at the beginning and then leave it as normal text?